cloud/deployment/deployment_variable.go

```diff
@@ -45,11 +49,11 @@ func VariableList(deploymentID, variableKey, ws, envFile, deploymentName string,
 	for i := range environmentVariablesObjects {
 		if environmentVariablesObjects[i].Key == variableKey {
 			nbEnvVarFound++
-			varTab.AddRow([]string{strconv.Itoa(nbEnvVarFound), environmentVariablesObjects[i].Key, environmentVariablesObjects[i].Value, strconv.FormatBool(environmentVariablesObjects[i].IsSecret)}, false)
+			varTab.AddRow([]string{strconv.Itoa(nbEnvVarFound), environmentVariablesObjects[i].Key, *environmentVariablesObjects[i].Value, strconv.FormatBool(environmentVariablesObjects[i].IsSecret)}, false)
 			break
 		} else if variableKey == "" {
 			nbEnvVarFound++
-			varTab.AddRow([]string{strconv.Itoa(nbEnvVarFound), environmentVariablesObjects[i].Key, environmentVariablesObjects[i].Value, strconv.FormatBool(environmentVariablesObjects[i].IsSecret)}, false)
+			varTab.AddRow([]string{strconv.Itoa(nbEnvVarFound), environmentVariablesObjects[i].Key, *environmentVariablesObjects[i].Value, strconv.FormatBool(environmentVariablesObjects[i].IsSecret)}, false)
```

Comment:
nit: i don't think the `*` is needed since we already dereferenced the struct slice in line 37

Reply:
for some reason environmentVariablesObjects[i].Value also needs to be deferred. If I don't do that it prints the pointer not the value